orchestra/tests/helpers/fixtures.py

```diff
@@ -399,24 +407,69 @@ def _setup_tasks(test_case, tasks):
     test_case.test_step = test_case.workflow_steps[
         test_case.test_version_slug][test_case.test_step_slug]
     for task_slug, details in tasks.items():
+        task_pickup_time = BASE_DATETIME + timedelta(hours=1)
         task = TaskFactory(
             project=test_case.projects[details['project_name']],
             step=test_case.test_step,
             status=details['status'],
-            start_datetime=parse('2015-10-12T01:00:00+00:00'),
+            start_datetime=task_pickup_time,
         )
         test_case.tasks[task_slug] = task
         for i, (user_id,
                 task_data,
                 assignment_status) in enumerate(details['assignments']):
-            TaskAssignmentFactory(
+            assignment = TaskAssignmentFactory(
                 worker=test_case.workers[user_id],
                 task=task,
                 status=assignment_status,
                 assignment_counter=i,
                 in_progress_task_data=task_data,
-                start_datetime=parse(
-                    '2015-10-12T0{}:00:00+00:00'.format(2 + i)))
+                start_datetime=_new_assignment_start_datetime(task))
+
+            # Each assignment must have at least one corresponding iteration
+            Iteration.objects.create(
```

Comment:
Create iterations for our fixture tasks, rethinking timing logic for all fixtures along the way